.github/workflows/module_7.yml

```diff
@@ -7,5 +7,5 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - run: docker build --target test --tag todo-app:test .
+      - run: docker build --platform linux/amd64 --target test --tag todo-app:test .
```

Comment:
Is this change needed (since GitHub runners would default to this architecture anyway)?